script.js

```diff
@@ -1,12 +1,98 @@
 //You can edit ALL of the code here
+
+const rootElem = document.getElementById("root");
+const inputSearch = document.querySelector("#liveSearch");
+const pCounter = document.querySelector("#counter");
+const selectEpisodes = document.querySelector("#selectEpisodes");
+const selectShow = document.querySelector("#selectShows");
+
 function setup() {
-  const allEpisodes = getAllEpisodes();
-  makePageForEpisodes(allEpisodes);
+  const allShows = getAllShows();
+  makeSelectShows(allShows); //level 400
+  fetchingShows();
+}
+
+function makeSelectShows(shows) {
+  shows.forEach((eachShow) => {
+    const option = document.createElement("option");
```

Comment:
Can you think of the scope of variable `option`? You've defined this variable inside your `loop`. How about if would have defined in global scope and re-use it in the other function?